handbook/engineering/code-insights/index.md

```diff
@@ -17,7 +17,7 @@ We go beyond single-step code intelligence and search to connect the full cycle
 
 Code insights is the first feature primarily built for non-search-based user personas (developers), instead focusing first on the needs of engineering directors and VPs.
 
-For more information about code insights, see the original [product document](https://docs.google.com/document/d/1EHzor6I1GhVVIpl70mH-c10b1tNEl_p1xRMJ9qHQfoc/edit) or this [demo](https://www.youtube.com/watch?v=XqeRb6Mc4Co) of a code insights prototype.
+For more information about code insights, see the original [product document](https://docs.google.com/document/d/1EHzor6I1GhVVIpl70mH-c10b1tNEl_p1xRMJ9qHQfoc/edit) or this [demo](https://www.youtube.com/watch?v=XqeRb6Mc4Co) of a code insights prototype. The Sourcegraph team can view the [quickstart guide](https://gist.github.com/Joelkw/f0582b164578aabc3ac936dee43f23e0) if they want to create their own.
```

Comment:
```suggestion
For more information about code insights, see the original [product document](https://docs.google.com/document/d/1EHzor6I1GhVVIpl70mH-c10b1tNEl_p1xRMJ9qHQfoc/edit) or this [demo](https://www.youtube.com/watch?v=XqeRb6Mc4Co) of a code insights prototype. If you want to create your own, you can view the [quickstart documentation](https://gist.github.com/Joelkw/f0582b164578aabc3ac936dee43f23e0), which is explicitly not in the Sourcegraph docs because insights is a prototype undergoing rapid development and the setup will change. 
```

Want to pre-empt any question here. Thanks for making this add!